### DISCUSSION TITLE
selectRowListForSelect с условием

### DISCUSSION THREAD

**Discussion by Alexmeb** [totumonline/totum-ru-issues-and-discussions#183] [Как это сделать?] [answered]
Циклическая таблица. Ряд пользователей должен видеть не все поля, которые возвращает selectRowListForSelect.
Но при этом через кнопку внутри цикла они должны установить этому полю то значение, которое не могут выбрать через selectRowListForSelect в циклической. Если скрыть от них эти поля, имеем зачеркнутое значение поля. Как бы это победить?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
https://ru.docs.totum.online/crossed-out-selects-and-what-it-may-be-related-to

В отображения Cелекта добавляется текущее значение:

```
=: selectRowListForSelect(table: 'table'; field: 'name'; where: 'id' = $listAdd)

listAdd: listAdd(list: $list; item: $#nfv)
    list: selectList(table: 'table'; field: 'id'; where: 'condition' = #condition_field)
```
